.prettierignore

```diff
@@ -11,3 +11,10 @@ node_modules
 pnpm-lock.yaml
 package-lock.json
 yarn.lock
+
+# Ignore files for typesafe-i18n auto-generated files
+src/i18n/i18n-svelte.ts
+src/i18n/i18n-types.ts
+src/i18n/i18n-util.async.ts
+src/i18n/i18n-util.sync.ts
+src/i18n/i18n-util.ts
```

Comment:
Prettier was auto-formatting these, but they will be re-generated every time we add/remove keys so we need to ignore them.